bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/GameEnd.java

```diff
@@ -0,0 +1,41 @@
+package com.andrei1058.bedwars.listeners;
+
+import com.andrei1058.bedwars.api.arena.generator.IGenerator;
+import com.andrei1058.bedwars.api.events.gameplay.GameEndEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.UUID;
+
+public class GameEnd implements Listener {
+
+    @EventHandler
+    public void stopPickUP(GameEndEvent event){
+        if (!event.getAliveWinners().isEmpty()){
+            for (UUID p : event.getAliveWinners()) {
+                Bukkit.getPlayer(p).setCanPickupItems(false);
```

Comment:
I'm unsure that would work on all mc versions

Reply:
I had people test it and it seems to work fine on all versions at least I know it works on 1.8.8 and 1.17

Reply:
I have addressed this issue